### PR TITLE
fix: inconsistent graphs layout

### LINKED ISSUE #718: [Bug] inconsist location of racing bar in perceptor tab on different repositories
### Version

v1.9.0

### Current Behavior

When open perceptor tab on small scale repos, the racing bar would be the first graph. While on large scale repos, it would be the last one.
image
image

### Expected Behavior

The order of the graphs on perceptor tab should be fixed.

### Environment

```markdown
- OS: MacOS
- Browser: Chrome
```

### Any additional comments?

This issue can be reproduced when this is the first time opening a large repo.

### PR DESCRIPTION
**Pull request by wxharry**
## Brief Information

This pull request is in the type of ([more info](https://github.com/angular/angular/blob/main/CONTRIBUTING.md#type) about types):

- [ ] build
- [ ] ci
- [ ] docs
- [ ] feat
- [x] fix
- [ ] perf
- [ ] refactor
- [ ] test

Related issues ([all available keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)):

- Resolve #718 

## Details
1. Add `div` elements in the `perceptor-layout` 
2. Update where we add the graphs

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/hypertrons/hypertrons-crx/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/hypertrons/hypertrons-crx)

### CODE REVIEW

File: src/pages/ContentScripts/features/perceptor-layout/view.tsx [hunk 1]
```diff
@@ -1,7 +1,12 @@
 import React from 'react';
 
 const View = (): JSX.Element => {
-  return <></>;
+  return (
+    <>
+      <div id="hypercrx-repo-activity-racing-bar" />
+      <div id="hypercrx-repo-networks" />
+    </>
+  );
```
Comment (tyn1998) [inline src/pages/ContentScripts/features/perceptor-layout/view.tsx:9]: There is already an element whose id is "hypercrx-repo-activity-racing-bar":

image

**Review by tyn1998 — CHANGES_REQUESTED** [review]
Hello @wxharry, your solution works but has an small issue. I suggest changing the ids to `#hypercrx-perceptor-slot-1`, `#hypercrx-perceptor-slot-2` or other names.

**Comment by tyn1998** [comment]
Also kindly remind you that the conventional commits [suggets](https://www.conventionalcommits.org/en/v1.0.0/#are-the-types-in-the-commit-title-uppercase-or-lowercase) using lowercase types :)

image

**Comment by wxharry** [comment]
> Also kindly remind you that the conventional commits [suggets](https://www.conventionalcommits.org/en/v1.0.0/#are-the-types-in-the-commit-title-uppercase-or-lowercase) using lowercase types :)
> 
> image

My bad. I have changed the PR title since we will squash all commits in this PR and use the title of this PR as the commit message.

File: src/pages/ContentScripts/features/perceptor-layout/view.tsx (hunk 1 above)
Comment (wxharry) [reply src/pages/ContentScripts/features/perceptor-layout/view.tsx]: You are right! I changed that to `#hypercrx-perceptor-slot-{feature-name}`.

Comment (tyn1998) [reply src/pages/ContentScripts/features/perceptor-layout/view.tsx]: Oops

image

File: src/pages/ContentScripts/features/repo-activity-racing-bar/index.tsx [hunk 2]
```diff
@@ -38,7 +38,7 @@ const init = async (): Promise<void> => {
   const container = document.createElement('div');
   container.id = featureId;
   // append before render so that the container has computed width
-  $('#hypercrx-perceptor-layout').append(container);
+  $('#hypercrx-repo-activity-racing-bar').append(container);
```
Comment (tyn1998) [inline src/pages/ContentScripts/features/repo-activity-racing-bar/index.tsx:41]: -> `$(`#hypercrx-perceptor-slot-xxx`).append(container);`

**Review by tyn1998 — APPROVED** [review]
It works, LGTM!

@wxharry Thank you so much

**Pull request merged**